dbt-cta/google_directory/models/1_cta_full_refresh/group_members.sql

```diff
@@ -0,0 +1,26 @@
+{% set partitions_to_replace = [
+    'timestamp_trunc(current_timestamp, day)',
+    'timestamp_trunc(timestamp_sub(current_timestamp, interval 1 day), day)'
+] %}
+
+{{ config(
+    cluster_by = "_airbyte_emitted_at",
```

Comment:
These partition replacement tables need the following config added too!

```
partitions = partitions_to_replace,
```

Reply:
oh dang, those are missing from the stripe models too! I'll make a separate PR to add those in...

Reply:
OK, just added `partitions=partitions_to_replace` in all the google_directory CTA base model configs!

Reply:
sweet thanks!